src/main/groovy/org/mockito/release/internal/gradle/GitPlugin.java

```diff
@@ -39,22 +39,29 @@ public void apply(final Project project) {
         TaskMaker.execTask(project, COMMIT_TASK, new Action<Exec>() {
             public void execute(final Exec t) {
                 t.setDescription("Commits staged changes using generic --author");
-                t.doFirst(new Action<Task>() {
-                    public void execute(Task task) {
-                        //doFirst (execution time) to pick up user-configured setting
+                lazyConfiguration(t, new Runnable() {
```

Comment:
Can you use deferredConfiguration here instead of lazyConfiguration? Please take a look at the javadocs for deferredConfiguration & lazyConfiguration. I will add a TODO for myself to explain when we should use lazyConfiguration and when we should use deferredConfiguration.